rulesdoc.md

```diff
@@ -378,45 +396,49 @@ Finally, choose 2 of the following bonuses.
 - Learn 1 Feat
 - Learn a maneuver and gain 2 proficiency with a weapon type
 - Increase your max wounds by 1
+- Increase your spell power by 5 and gain 3 mana
 
 ##### Variant Rule - Rolling for Attributes
 Instead of using the rules above for attributes, your group may decide to roll your scores. For each attribute roll 1 Superior, 3 Normal, and 1 Terrible die, adding up the total successes. The minimum score for an attribute is -2, with any rolls below that counting as -2 for the roll.
 
 ### Starting Equipment.
-Depending on your background you may start with additional gear. All players start with an adventuring kit containing a bedroll, 10 days of rations, a fire starter and 50ft of rope. In addition you start with 50 gold which you can spend on gear or upgrades. Any unspent gold becomes your characters starting money.
+Depending on your background you may start with additional gear. All players start with an adventuring kit containing a bedroll, 10 days of rations, a fire starter and 50ft of rope. In addition you start with 50 silver which you can spend on gear or upgrades. Any unspent silver becomes your characters starting money.
+
+#### Currency
```

Comment:
Using the D&D money is kinda weird as copper pieces are basically useless, and the difference between coins is low enough that it's hard to get a good idea of value in some cases. (is 100 gold a lot? 10 plat? do poor people have plat if it's only 10 gold?)

With this change, copper pieces are replaced by the copper trite. Your silver will be converted to 10 copper trites each. Gold converts to silver crosses at a 1:1 ratio. Platinum becomes platinum talents at a 10:1 ratio.

This should give a better scale to things. 1 copper trite is roughly the same as 10-25cents. A meal or drink would be anywhere from 5-15ct, or a stay at an inn costing 25-50ct. Silver cross is ~10-20 dollars. a days wage for a commoner could be 1-3 silver crosses. Buying an animal being 5-15sc, armor and weapons costing 10-100sc. Platinum talents are larger rectangular tokens that would fit in the palm of your hand like a bill. These are roughly the same as 1000$ and are used for business contracts, transferring money, buying magic items and the like. Payment for a quest could be in 1-3 platinum talents. Buying a building, such as a tavern, would be anywhere from 50-100 talents, common magic items could be as low as 3-5 talents, or rarer/more powerful going upwards of 25.

This is great for me as Each currency has a set purpose and I don't have to come up with values on the spot that 'seem reasonable' but have no consistency between rewards. (remember getting 50 gold for the first quest in hawthorne way back when, and then getting 300 from thornbrook was kinda meh?)

Reply:
This has better flavor, but in the end you need to decide if you are just loosely keeping track of wealth, or you are trying to add some scarcity or RP scenarios. Can you carry more than 100 coins? Do you need to deposit money at Valint & Balk and get letters of credit? In a lot of fantasy there is perceptual inflation, but here's a well-researched segment on wages:

The basic wage of the mercenary German common foot soldier throughout the 16th century, four florins per month, compared favorably to that received by other workers. Day laborers in towns earned no more than two and a half florins per month while in the 1540's, a male agricultural worker received, besides food and lodging, three to ten and a half florins per year (Redlich 1964 vol. i, pp. 122, 127). Mercenaries could, of course, supplement their income by looting. By contract, they were permitted to keep harnesses, armor, and other movable objects. During the mid-17th century, for example, at a time when a skilled worker in Germany might possess wealth of 100 florins, a soldier returned from his very first campaign with booty worth 200 florins (Redlich 1956, p. 54). Some soldiers accumulated 8,000 to 20,000 talers by looting (Redlich 1964, vol. i, p. 496). Mercenaries could also ransom prisoners of war.2 Around 1600, it was fair to require as ransom the annual income of the prisoner's property or his annual pay if he did not possess property (Redlich 1956, p. 33).

Reply:
Part of it is so I have a clear distinction of wealth: Trites for basic/low cost things, Crosses for most common purchases, Talents for rich people, high end deals, buildings, etc. There is some RP that could be done there, for example someone coming in and dropping 10 talents on the table for you to complete a job feels more impactful than 1000 crosses. They're far enough apart that I'd be less likely to hand wave converting wealth between each coin but in general that level of micromanagement becomes tedious quickly. We haven't dealt with weight much before, but I could see getting a large number of Trites or Crosses and having to find a way to transport it/convert it so you can actually carry it all with you.